src/StickyParallaxHeader.js

```diff
@@ -29,6 +29,35 @@ import { getSafelyScrollNode, setRef } from './utils';
 const { divide, Value, createAnimatedComponent, event, timing, ValueXY } = Animated;
 const AnimatedScrollView = createAnimatedComponent(ScrollView);
 
+const HeaderBackgroundImage = (props) => {
+  const { backgroundHeight, backgroundImage, background } = props;
+  const AnimatedImageBackground = createAnimatedComponent(ImageBackground);
+
+  return (
+    <AnimatedImageBackground
+      style={[
+        styles.headerStyle,
+        {
+          height: backgroundHeight,
+        },
+      ]}
+      source={backgroundImage}>
+      {background}
+    </AnimatedImageBackground>
+  );
+};
+
+HeaderBackgroundImage.propTypes = {
+  background: node,
+  backgroundHeight: number,
+  backgroundImage: Image.propTypes.source,
+};
+
+const headerImagesAreEqual = (prevProps, props) =>
+  prevProps.backgroundImage.uri === props.backgroundImage.uri;
```

Comment:
Don't we need to add conditions for other props as well?
If `backgroundHeight` or `background` prop changes, the component still needs to rerender.